### DISCUSSION TITLE
Addition of this repository for Google Summer of Code

### DISCUSSION THREAD

**Discussion by AbhijithGanesh** [CiviWiki/OpenCiviWiki#1143] [Ideas]
According to the summer of [code](https://summerofcode.withgoogle.com/), open sourced repos, can be listed there to increase participation from new members,

Their registration date opens on January 30,2022

Further details can be found [here](https://summerofcode.withgoogle.com/how-it-works/)

**Comment by AbhijithGanesh** [comment]
@brylie and @gorkemarslan what do you think about this?

**Reply by brylie** [reply]
It seems like a good idea.

**Reply by GokuMizuno** [reply]
I will agree with Brylie on this one.  Very good idea.

**Reply by gorkemarslan** [reply]
Sounds great. :)

**Reply by AbhijithGanesh** [reply]
Great, I will send a reminder again. I shall add their links soon

**Comment by AbhijithGanesh** [comment]
@brylie and @gorkemarslan I think GSoc is open for registrations, can you look into it please.
